live-examples/js-examples/array-sort.html

```diff
@@ -0,0 +1,16 @@
+<pre>
+<code id="static-js">var array1 = ['March', 'Jan', 'Feb', 'Dev'];
+
+array1.sort();
+
```

Comment:
I think we should remove this blank line too - then we will fit in the editor without needing a scrollbar.